<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemStore</class>
 <widget class="QWidget" name="SystemStore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buy">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>580</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>购买</string>
   </property>
  </widget>
  <widget class="QLabel" name="point">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>系统点数:9223372036854775807</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemName">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>400</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>物品名称</string>
   </property>
  </widget>
  <widget class="QListWidget" name="stoneList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>15</y>
     <width>400</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="describe">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>15</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;物品描述&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="itemFunction">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;物品作用&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="itemType">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>物品类别</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="count">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QLabel" name="money">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>430</y>
     <width>400</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>所花点数:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
